### PR TITLE
Fix Python packages nevow and magic-wormhole-mailbox-server

### PR DESCRIPTION
**Pull request by rycee**
To get a bit closer to an unbroken tahoe-lafs package…

### CODE REVIEW

File: pkgs/development/python-modules/nevow/default.nix [hunk 1]
```diff
@@ -10,6 +10,14 @@ buildPythonPackage rec {
     sha256 = "2299a0d2a0c1312040705599d5d571acfea74df82b968c0b9264f6f45266cf6e";
   };
 
+  patches = [
+    # Fix builds against recent Twisted.
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/twisted/nevow/pull/109.patch";
```
Comment (marsam) [inline pkgs/development/python-modules/nevow/default.nix]: please use the commit patch url
```suggestion
      url = "https://github.com/twisted/nevow/commit/f1b366f1a73009b6a1df12fa6f4dc464c564c944.patch";
```

**Review by jonringer — APPROVED** [review]
diff LGTM (after @marsam suggestion is applied)
commits LGTM

```
[9 built, 97 copied (269.5 MiB), 73.5 MiB DL]
https://github.com/NixOS/nixpkgs/pull/75899
2 package are marked as broken and were skipped:
python27Packages.magic-wormhole tahoe-lafs

7 package were built:
gnome-keysign magic-wormhole python27Packages.magic-wormhole-mailbox-server python27Packages.nevow python37Packages.magic-wormhole-mailbox-server python38Packages.magic-wormhole python38Packages.magic-wormhole-mailbox-server
```

File: pkgs/development/python-modules/nevow/default.nix (hunk 1 above)
Comment (rycee) [reply pkgs/development/python-modules/nevow/default.nix]: Good point. Should be fixed now.

**Review by jonringer — APPROVED** [review]
still LGTM

```
[56 built, 336 copied (1067.1 MiB), 288.9 MiB DL]
https://github.com/NixOS/nixpkgs/pull/75899
7 package were built:
gnome-keysign magic-wormhole python27Packages.magic-wormhole-mailbox-server python27Packages.nevow python37Packages.magic-wormhole-mailbox-server python38Packages.magic-wormhole python38Packages.magic-wormhole-mailbox-server
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build gnome-keysign magic-wormhole python27Packages.magic-wormhole-mailbox-server python27Packages.nevow python37Packages.magic-wormhole-mailbox-server python38Packages.magic-wormhole python38Packages.magic-wormhole-mailbox-server

**Comment by jonringer** [comment]
I built it on x64, arch is having problems on master right now, built successfully for darwin

**Pull request merged**